tutorials/abap-extensibility-cbo-create/abap-extensibility-cbo-create.md

```diff
@@ -38,9 +34,9 @@ In the first parts a manager wants to define business objects "Bonus Plan" for e
 
 1. Enter your SAP S/4HANA system's Fiori Launchpad.
 
-2. Open application **Custom Business Objects** from **Extensibility** Launchpad group.
+2. Start typing **Custom Business Objects** in the Launchpad search and open the App from the results.
 
-    ![Custom Business Objects application tile](tile_CBO.png)
+    ![Custom Business Objects application from search results](FLP_search_resultCBO.png)
 
 
 ### Create a custom business object
```

Comment:
Press new to start the creation of a custom business object.

The other editable fields will be automatically filled up with suggested values, but can be changed.